libraries/core/commands/plotProjects.js

```diff
@@ -0,0 +1,28 @@
+import AppCommand from '../classes/AppCommand';
+
+/**
+ * Enables the geofencing provider plotProjects
+ * @param {string} token The public token that is needed to enable plot projects
+ */
+export const plotProjectsEnable = (token) => {
+  const command = new AppCommand();
+
+  command
```

Comment:
Please provide simple tests for the two commands.

Reply:
done